DecipherNow/gm-fabric-dashboard
# fix: 1015 confirmation modal is pushed up too high in FF

Resolves #1015 

## Proposed Changes

  - assigned 100vh to confirmation modal container div created by React-Modal
  
## How to Test

  - Settings > click on `Clear Cache` button at the bottom and confirm that modal is not pushed up too high in all supported versions of Firefox browser
